packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -12,6 +12,7 @@ import 'dart:math';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/scheduler.dart';
 
+import '../painting/basic_types.dart';
```

Comment:
Sorry, one more thing I completely glossed over: we usually do relative imports for files within the same package, and package imports otherwise.

```suggestion
import 'package:flutter/painting.dart' show Axis;
import 'package:flutter/scheduler.dart';

```

Reply:
Updated.